pkg/apis/kops/dockerconfig.go

```diff
@@ -50,6 +50,8 @@ type DockerConfig struct {
 	Storage *string `json:"storage,omitempty" flag:"storage-driver"`
 	// StorageOpts is a series of options passed to the storage driver
 	StorageOpts []string `json:"storageOpts,omitempty" flag:"storage-opt,repeat"`
+	// UserspaceRemap sets the userspace remapping option for the docker daemon
+	UserspaceRemap string `json:"userspaceRemap,omitempty" flag:"userns-remap"`
```

Comment:
Can we name the field `UserNamespaceRemap` and json `userNamespaceRemap`?  Userspace means something else to me :-)

Reply:
yep ... make's sense! :-)